layouts/_default/article.html

```diff
@@ -1,19 +1,24 @@
-<div class="article">
-    <img src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl">
-    <div class="bg-[#ECF6FF] rounded-b-xl">
-        <div class="px-6 pt-4">
+<div class="article ">
+    <img src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl" style="height:212px; width:100%">
+    <div class="bg-[#ECF6FF] rounded-b-xl ">
+        <div class="px-6 pt-4h  ">
```

Comment:
```suggestion
        <div class="px-6 pt-4 ">
```

Reply:
I thought this might be a typo as I didn't see any Tailwind classes for padding that end with "h". If it's not, then you may ignore the suggestion.